src/js/modules/rescheme.js

```diff
@@ -79,23 +79,37 @@ class Rescheme {
     * @param {boolean} arg3 animatetransition: If the transition should be animated
     */
   static changeTheme(arg1, arg2 = true, arg3 = true) {
```

Comment:
This whole function shall be documented from inside, and/or split into multiple sub-functions. It is quite big and hardly readable